wotlk_cat_sim.py

```diff
@@ -358,6 +358,7 @@ def rake(self, time):
             self.rake_end = time + self.player.rake_duration
             self.rake_ticks = list(np.arange(time + 3, self.rake_end + 1e-9, 3))
             self.rake_damage = self.player.rake_tick
+            self.rake_crit_chance = self.player.crit_chance
```

Comment:
snapshot the crit chance when the debuff is applied, not sure if any other bonus